app/code/Magento/QuoteGraphQl/Model/Resolver/SetShippingMethodsOnCart.php

```diff
@@ -69,13 +69,13 @@ public function resolve(Field $field, $context, ResolveInfo $info, array $value
 
         $shippingMethod = reset($shippingMethods); // This point can be extended for multishipping
 
-        if (!$shippingMethod['cart_address_id']) {
+        if (!array_key_exists('cart_address_id', $shippingMethod)) {
```

Comment:
We can replace with `isset` and `empty`
Please check, why we don't have notices during test running

Reply:
used 'isset' instead array_key_exists